spec/models/reference_spec.rb

```diff
@@ -67,4 +67,28 @@
       end
     end
   end
+
+  describe 'auditing' do
```

Comment:
[The RSpec matchers](https://www.rubydoc.info/gems/audited/4.4.1/RSpec/Matchers) that come baked into `audited` may slim these specs down?

Reply:
See https://github.com/DFE-Digital/manage-courses-backend/pull/259/files for examples

Reply:
thanks! I've used those to replace 2 of the 3 cases, I think the third probably still needs testing explicitly